.github/workflows/test_workflow.yml

```diff
@@ -565,27 +565,35 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: "11"
+      - name: Cache JDK Downloads
+        uses: actions/cache@v3
+        with:
+          path: /home/runner/jvm
+          key: ${{ runner.os }}-zing-jdk-${{ matrix.java_version }}-v0
```

Comment:
@jbachorik any reason not to add caching to the JDK downloads ?
I would need to adjust this to all the jobs

Reply:
Caching the jdk binaries would be useful. We would key on the distro/version/arch but that should speed up builds significantly.